docs/specs/toc.yml

```
@@ -1605,3 +1605,109 @@ outputs:
         "a.md": [{ "source": "TOC.yml", "type": "metadata" }]
       }
     }
+---
+# TOC split metadata supporting
+repos:
+  https://ms.com/ops-file-metadata#master:
+  - files:
+      docfx.yml:
+      .openpublishing.publish.config.json: |
+        {
+          "JoinTOCPlugin": [
+            {
+              "ConceptualTOC": "b/TOC.yml",
+              "ReferenceTOCUrl": "/dotnet-test/api/toc.json",
+              "ConceptualTOCUrl": "dotnet-test/learn/toc.json",
+              "ReferenceTOC": "a/TOC.yml"
+            }
+          ]
+        }
+      a/TOC.yml: |
+        splitItemsBy: name
+        items:
+        - name: a
+          items:
+          - name: a-a
+        - name: b
+          items:
+          - name: b-b
+      b/TOC.yml: |
+        splitItemsBy: name
+        items:
+        - name: 1
+          items:
+          - name: 1-1
+outputs:
+  b/toc.json: |
+    {
+      "metadata": {
+        "universal_ref_toc": "/dotnet-test/api/toc.json"
+      },
+      "items": [
+        {
```

nit: consider compact where appropriate to fit the test case inside a screen (makes it easier to read without scrolling up and down)
```suggestion
        { "name": "1" }
```